via/views/route_by_content.py

```diff
@@ -4,15 +4,21 @@
 from pyramid import httpexceptions as exc
 from pyramid import view
 
-from via.services import URLDetailsService, ViaClientService, has_secure_url_token
+from via.services import (
+    CheckmateService,
+    URLDetailsService,
+    ViaClientService,
+    has_secure_url_token,
+)
 
 
 @view.view_config(route_name="route_by_content", decorator=(has_secure_url_token,))
 def route_by_content(context, request):
     """Routes the request according to the Content-Type header."""
     url = context.url_from_query()
+    checkmate_service = request.find_service(CheckmateService)
 
-    request.checkmate.raise_if_blocked(url)
+    checkmate_service.raise_if_blocked(url)
```

Comment:
Similar suggestion to above, but I can't do it in GH because of the removed line.

Reply:
:+1: https://github.com/hypothesis/via/pull/1004/commits/131a64b66c923a4a8a4e53c56f06e38c84e592e3